### PR TITLE
LBSD-1370 Limit number of characters for the photographer field to ma…

### PR DESCRIPTION
**Pull request by vladnicoara**
…x. 300

### CODE REVIEW

File: client/app/scripts/ng-sir-trevor-blocks.js [hunk 1]
```diff
@@ -438,12 +439,15 @@ define([
...
-                    
+
                     this.$editor.append($('<div>', {
                         name: 'credit',
                         class: 'st-image-block',
                         contenteditable: true,
-                        placeholder: that.authorPlaceholder
+                        placeholder: that.authorPlaceholder,
+                        oninput:
+                        'if(this.innerText.length > 300) { alert("' + that.limitedText + '"); this.innerText = this.innerText.substring(0, 300); this.style="border:1px solid red"}' +
```
Comment (m0g) [inline client/app/scripts/ng-sir-trevor-blocks.js]: What's going on here?

Comment (vladnicoara) [inline client/app/scripts/ng-sir-trevor-blocks.js]: Adding 'oninput' attribute for the image author. oninput works for paste as well as opposed to keyup or keydown. As for the location and the form of the code, I did not find a better way to work with this sir trevor element.

Comment (m0g) [inline client/app/scripts/ng-sir-trevor-blocks.js]: Yes, but do you really need to inject javascript code like that?

Comment (vladnicoara) [inline client/app/scripts/ng-sir-trevor-blocks.js]: That is the first thing I have tried of course. If I define a function there( when appending the input to the editor) or pass a defined one, it just gets called once on render and then nothing.
I found a way to add that functionality in proper way after it has been rendered

**Pull request merged**